### PR TITLE
Update user guide to reflect list of pre-defined constants for

### PR DESCRIPTION
**Pull request by cwq2326**
Updated the add command section in the user guide to tell users that they must use pre-defined constants for fields that contain enum class such as faculty and covid status. Also included the list of available pre-defined constants in the table.

Updated the developer guide under the use case of adding a student into Tracey to include an extension whereby the user key in a wrong pre-defined constants for the faculty and covid status tags.

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -392,6 +392,12 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
         Use case ends
 * 1c. User adds multiple students in one go.
     * 1c1. Tracey will list out a list of new students added with their info.
+
+* 1d. User uses wrong pre-defined constants for fields such as faculty or covid status.
+
+    * 1d1. Tracey will provide a list of pre-defined constants for the user.
+
+    * 1d2. User use the correct pre-defined constants for the respective tags.
```
Comment (tanchinkiat99) [inline docs/DeveloperGuide.md:400]: I think this is the correct flow!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -183,7 +184,7 @@ of the data (Covid statuses) and give the user an easier understanding of how th
 | No. | Command | Description |
 | - | - | - |
 | 1. | `find` | Included in `AB3`|
-| 2. | `add' | Included in AB3, Additional Field: Include health categories (Positive, Close Contact, Transient Contact, Health Risk Warning etc.) |
+| 2. | `add' | Included in AB3 with additional attributes such as faculty, matriculation number and covid status |
```
Comment (tanchinkiat99) [inline docs/UserGuide.md:187]: Yep, I think this sums up our changes to the Add function.

**Review by tanchinkiat99 — COMMENTED** [review]
LGTM! I think this is ready to be merged, good effort! 👍

**Review by jiale-c — APPROVED** [review]
LGTM!

**Pull request merged**